index.html

```diff
@@ -431,112 +363,144 @@ <h2 class="connect">let's connect</h2>
+              <ul class="lang-list">
+                <li class="js-lang langs">
+                  <img
+                    class="lang-img"
+                    src="./assets/js.png"
+                    alt="javascript"
+                  />
+                  <p>Javascript</p>
+                </li>
+                <li class="html-lang langs">
+                  <img
+                    class="lang-img"
+                    src="./assets/html.png"
+                    alt="javascript"
+                  />
+                  <p>HTML</p>
+                </li>
+                <li class="css-lang langs">
+                  <img
+                    class="lang-img"
+                    src="./assets/css.png"
+                    alt="javascript"
+                  />
+                  <p>CSS</p>
+                </li>
+              </ul>
+            </div>
           </div>
         </div>
       </section>
-      <section class="contact">
-        <div class="contact-me">
-          <h1>Contact Me</h1>
-          <p>
-            If you have an application you are interested in developing, a
-            feature that you need built or a project that needs coding. I’d love
-            to help with it
-          </p>
+      <section class="contact-background">
+        <div class="contact">
+          <div class="contact-me">
+            <h1>Contact Me</h1>
+            <p>
+              If you have an application you are interested in developing, a
+              feature that you need built or a project that needs coding. I’d
+              love to help with it
+            </p>
+            <form action="https://formspree.io/f/xpzkbobk" method="POST">
+              <ul>
+                <li>
+                  <label for="names" class="invisible">Names</label>
+                  <input
+                    type="text"
+                    name="names"
+                    maxlength="30"
+                    placeholder="Full name"
+                    id="names"
+                    required
+                  />
+                </li>
+                <li>
+                  <label for="email" class="invisible">Email</label>
+                  <input
+                    type="email"
+                    name="email"
+                    placeholder="Email"
+                    id="email"
+                    required
+                  />
+                </li>
+                <li id="message-area">
+                  <label for="message" class="invisible"></label>
+                  <textarea
+                    name="message"
+                    id="message"
+                    placeholder="Write your message here"
+                    cols="30"
+                    maxlength="500"
+                    rows="10"
+                    required
+                  ></textarea>
+                </li>
+              </ul>
+              <button class="send-message" type="submit">Get in touch</button>
```

Comment:
- [ ] Please, center the button in your "Contact me" section.

![image](https://user-images.githubusercontent.com/60164239/134586820-58f0a9ef-2888-44b0-9c92-fcdaba439d1c.png)

Reply:
- [ ] The issue here has not been properly resolved as the button is still aligned to the right.
- [ ] This issue has not been resolved properly "Improve the description in your Pull Request, explain what you did, what is the main scope, which files do you changed or added, what you set up in your project in this pull request, and so on."